packages/capnpc-ts/src/generators.ts

```diff
@@ -375,6 +383,8 @@ export function generateStructFieldMethods(
 
     default:
 
+      // TODO Maybe this should be an error?
```

Comment:
It should log a warning to `console.error`: "CAPNP-TS: Ignoring code generator request for unsupported struct field type: %s"